service_container/3.3-di-changes.rst

```diff
@@ -468,6 +470,54 @@ Many autoconfigured tags have an optional priority. If you need to specify a pri
 (or any other optional tag attribute), no problem! Just :ref:`manually configure your service <services-manually-wire-args>`
 and add the tag. Your tag will take precedence over the one added by auto-configuration.
 
+5) Auto-configure with _instanceof
+----------------------------------
+
+And the final big change is ``_instanceof``. It acts as a default definition
+template (see `service-33-default_definition`_), but only for services whose
+class matches a defined one.
+This can be very useful when many services share some tag that cannot be
```

Comment:
If you want to make this a new paragraph, there must be a blank line before it. Otherwise, I would not break lines here.

Reply:
You mean you would break them at this point of the sentence or you wouldn't break at all?

Reply:
Because the sf docs standards explicitely state lines should be broken when exceeding 72 chars